src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -463,10 +484,12 @@ public Void caseStringType() {
 
         @Override
         public Void caseObjectType(ObjectType type) {
-          if (type.isRecordType()) {
+          if (type.isNativeObjectType()) {
```

Comment:
NativeObjectType seems to be the builtin 'Object' type, as opposed to a
subtype of Object. but I don't really know

On Wed, Aug 26, 2015 at 1:05 PM Rado Kirov <notifications@github.com> wrote:

> In src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java
> <https://github.com/angular/closure-to-dts/pull/39#discussion_r38028221>:
>
> > @@ -463,10 +484,12 @@ public Void caseStringType() {
> >
> >          @Override
> >          public Void caseObjectType(ObjectType type) {
> > -          if (type.isRecordType()) {
> > +          if (type.isNativeObjectType()) {
>
> dumb question, what is the diff between NativeObjectType and ObjectType?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/closure-to-dts/pull/39/files#r38028221>.
>